.ci.yaml

```diff
@@ -905,7 +906,7 @@ targets:
           {"dependency": "vs_build", "version": "version:vs2019"}
         ]
 
-  - name: Windows dart_unit_tests_shard_1 master
+  - name: Windows_x64 dart_unit_tests_shard_1 master
```

Comment:
You can't just rename tasks unfortunately; ci.yaml doesn't allow for that workflow. You have to add a whole new duplicate set of tasks in bringup mode, and then do a second PR—after the first has both landed and propagated through the infra system—to remove the old ones and enable the new ones.

Reply:
Sounds good, will do!